### ISSUE TITLE
Editing causes Output panel to show and steal focus

### ISSUE THREAD

**Issue by anvish** [mrmlnc/vscode-stylefmt#84]
When adding a new rule in the middle of scss file, output panel opens and steals focus:
```
[Stylefmt]
CssSyntaxError: <file_path>:1:1: Unknown word

> 1 | <rule_name>
    | ^
```

**Comment by mrmlnc** [comment]
Please, provide your editor settings and steps to reproduce your problem. Will be cool if you provide GIF or video.

**Comment by anvish** [comment]
1. Copy class name
2. Open scss file
3. Press dot
4. Paste
5. Error window appears

![bug](https://cloud.githubusercontent.com/assets/1114090/23804494/2636d74e-05cb-11e7-9690-149590bb912e.gif)

**Comment by mrmlnc** [comment]
This is related to `editor.formatOnPaste` option. VS Code try to format code after paste, but since it is invalid code `stylefmt` throw error.

**Comment by mrmlnc** [comment]
Closing as we need more info to investigate this.

**Comment by evanwarner** [comment]
What additional info do you need? I'm getting the exact same problem.
```
"editor.formatOnType": false,
"editor.formatOnPaste": false,
"editor.formatOnSave": false,
```

Super annoying as I'm constantly writing HTML then copying classes into my stylesheet.

**Comment by mrmlnc** [comment]
I don't fully understand your problem. You write invalid CSS code and plugin gives you an error. You want option to disable error output?

**Comment by evanwarner** [comment]
Well, the code is invalid in the sense that I haven't finished typing yet, yes. But the error highlighter is not the problem (that's working fine). The problem unique to this case is that the output panel pops up and steals focus. That doesn't happen when I'm just writing a class name. It only happens when I paste a class name in.

![stylefmt-error](https://cloud.githubusercontent.com/assets/6453842/25099272/916baa82-2379-11e7-8285-b82e93afad7f.gif)

**Comment by mrmlnc** [comment]
Very strange, because this plugin attempts to format the code only when the command `stylefmt.execute` is executed or you use `Format` from editor.

You can provide your settings?

**Comment by evanwarner** [comment]
Ahhh, ok I found the culprit. I have the [multiple clipboards](https://marketplace.visualstudio.com/items?itemName=slevesque.vscode-multiclip) extension installed. I just set `"multiclip.formatAfterPaste": false` and problem solved!

@anvish Is this the same case for you?

**Comment by anvish** [comment]
@evanwarner no, I don't have this extension, I have "editor.formatOnPaste": true

@mrmlnc Problem with the panel is not solved. I think the issue should be in open state. Were you able to reproduce the issue? These steps reliably reproduce it for me ("editor.formatOnPaste" should be set to true):
* Copy some class name
* Close VSCode
* Open VSCode with css file
* Enter dot
* Press ctrl+v

Output panel pops up and steals the focus. This should not happen.

**Comment by mrmlnc** [comment]
I'll add `stylefmt.showError` option. By default this option will be enabled.

**Comment by mrmlnc** [comment]
Will be fixed in 2.3.0. Code in master. Release very soon.

**Comment by rollecode** [comment]
`stylefmt.showError` seems not to have any effect :(
